src/haproxy/haproxy.cfg

```diff
@@ -1,134 +1,190 @@
 global
-  tune.ssl.default-dh-param 1024
+	tune.ssl.default-dh-param 1024
```

Comment:
Can you please switch white-space to 2 spaces to make it easier to see the actual changes?